torch/_tensor_docs.py

```diff
@@ -5485,15 +5563,51 @@ def callable(a, b) -> number
     >>> sparse_bsr.col_indices()
     tensor([0, 1, 0, 1])
 
+    >>> dense = torch.zeros(4, 3, 1)
+    >>> dense[0:2, 0] = dense[0:2, 2] = dense[2:4, 1] = 1
+    >>> dense.to_sparse_bsr((2, 1), 1)
+    tensor(crow_indices=tensor([0, 2, 3]),
+           col_indices=tensor([0, 2, 1]),
+           values=tensor([[[[1.]],
+
+                           [[1.]]],
+
```

Comment:
Is this additional whitespace really how we print BSR Tensors?

@alexsamardzic - you can see the [preview of the Python documentation](https://docs-preview.pytorch.org/90177/generated/torch.Tensor.to_sparse_bsr.html?highlight=to_sparse_bsr#torch.Tensor.to_sparse_bsr) as part of the [Helpful Links](https://github.com/pytorch/pytorch/pull/90177#issuecomment-1337305137). You'll just need to [search for the function name that you changed](https://docs-preview.pytorch.org/90177/search.html?q=to_sparse_bsr&check_keywords=yes&area=default) to see how it renders on the web.

Reply:
@cpuhrsch printing BSR tensors are defined as how we print strided tensors. For example:
```python
>>> torch.ones(4, 4).to_sparse_bsr((2, 2)).values()
tensor([[[1., 1.],
         [1., 1.]],

        [[1., 1.],
         [1., 1.]],

        [[1., 1.],
         [1., 1.]],

        [[1., 1.],
         [1., 1.]]])
```

Personally, I also find the whitespace annoying but I guess this is supposed to help to view high-dimensional tensors.

Reply:
Indeed, I've copied output from the interpreter for these examples.